src/ImageSharp.Drawing/Processing/Processors/Drawing/DrawImageProcessor{TPixelBg,TPixelFg}.cs

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+
+using SixLabors.ImageSharp.Advanced;
+using SixLabors.ImageSharp.ParallelUtils;
+using SixLabors.ImageSharp.PixelFormats;
+using SixLabors.Primitives;
+
+namespace SixLabors.ImageSharp.Processing.Processors.Drawing
+{
+    /// <summary>
+    /// Combines two images together by blending the pixels.
+    /// </summary>
+    /// <typeparam name="TPixelBg">The pixel format of destination image.</typeparam>
+    /// <typeparam name="TPixelFg">The pixel format of source image.</typeparam>
+    internal class DrawImageProcessor<TPixelBg, TPixelFg> : ImageProcessor<TPixelBg>
```

Comment:
This is an interesting one. I personally find Fg and Bg far less confusing than the compositing standard of Src and Dst but others might argue the inconsistency in language causes confusion. 

They're wrong of course 😄

Reply:
That naming totally confused me, had to rename this in order to avoid making a mistake.